docs/painless/painless-contexts/painless-filter-context.asciidoc

```diff
@@ -23,4 +23,45 @@ query to include and exclude documents.
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+This script allows to find all documents where the value of `doc['sold']`
+is `false`, and where the cost is less than 18.
+
```

Comment:
Instead of "allows to find", I'd just say "finds". Given the straightforwardness of the example, I'd just summarize the intent of the script, instead of repeating what the code says:

This script finds all unsold documents that cost less than $18.